packages/shared/src/models/contributor/ContributorPersonalStats.ts

```diff
@@ -1,17 +1,18 @@
 import * as t from 'io-ts';
 import { StringOrNull } from '../common/StringOrNull';
 
-export const HomeMetadata = t.strict(
+export const HomeMetadata = t.partial(
```

Comment:
This one should remain a `t.strict` and use a `t.union([$Codec, t.undefined])` for _optional property_ (where `$Codec` could be one of `t.string, t.number, t.boolean, etc...`)